next/oracle/delete_um_tenant.sh

```
@@ -0,0 +1,82 @@
+for TENANT_ID in $COMMON_TENANTS; do
+  echo "Processing TENANT_ID=$TENANT_ID"
+
+  # Generate the delete procedure
+  DELETE_PROCEDURE="DELETE FROM UM_TENANT WHERE UM_ID = ${TENANT_ID};"
+
+  # Execute the delete procedure
+  echo "Executing deletion procedure for TENANT_ID=$TENANT_ID..."
+  echo "$DELETE_PROCEDURE" | sqlplus -s "$DB_USER/$DB_PASSWORD@//$DB_HOST:$DB_PORT/$DB_NAME" > /dev/null 2>&1
+
+  # Check if the execution was successful
+  if [[ $? -eq 0 ]]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)